core/frontend/src/components/version-chooser/VersionChooser.vue

```diff
@@ -9,6 +9,16 @@
         max-width="900"
         class="mx-auto my-12 pa-4"
       >
+        <v-alert
+          border="top"
+          colored-border
+          type="info"
+          elevation="2"
+        >
+          Pirate mode can be enabled to show all official unstable versions,
+          including beta and custom BlueOs releases from Blue Robotics.
```

Comment:
```suggestion
          Turn on Pirate mode to view all available BlueOS versions, including previous installs
          stored onboard, as well as past and current stable, beta, and development releases
          downloadable from Blue Robotics.
```

We may want to make "Turn on Pirate mode" into a hyperlink that turns it on, or that opens the thing that lets you turn it on.

Also, should this description still be shown if Pirate mode is already on? I'm undecided on my preference. We could potentially also toggle-change it to start with something like "Turn on Pirate mode to view" -> "This page shows" when Pirate mode is on.

Reply:
> Also, should this description still be shown if Pirate mode is already on?

There is a if on the parent component for it.



> We may want to make "Turn on Pirate mode" into a hyperlink that turns it on, or that opens the thing that lets you turn it on.

We can keep that in mind while we are refactoring the pirate-mode enable disable functionality, right now to do it we already need to rework the main component, so it's better to keep in a future PR of the main refactor.